source/containers.tex

```diff
@@ -2746,8 +2746,10 @@
 \tcode{b.cbegin(n)}
 &   \tcode{const_local_iterator}
 &   \requires \tcode{n} shall be in the range \tcode{[0, b.bucket_count())}.
-    Note: \tcode{[b.cbegin(n), b.cend(n))} is a valid range containing
-    all of the elements in the $\texttt{n}^{\textrm{ th}}$ bucket.%
+    \begin{note}
+    \tcode{[b.cbegin(n), b.cend(n))} is a valid range containing
+    all of the elements in the $\texttt{n}^{\textrm{ th}}$ bucket.
+    \end{note}%
```

Comment:
Where (if anywhere) is the normative rule justifying this claim? Should we copy the corresponding wording from `b.begin(n)` and `b.end(n)` to `b.cbegin(n)` and `b.cend(n)`, or is there general wording saying how `begin` and `cbegin` relate somewhere?

Reply:
For ordinary iterators and begin/end, you could squint at Table 83 to conclude this relation. But we never have *local* iterators anywhere. We don't have equivalent "operational semantics" fields in this table.

It might be safest to repeat the wording from the non-const versions and delete the note. What do you think?

Reply:
That approach sounds good to me.

Reply:
OK, done, PTAL.